src/Explorer/Panes/DeleteDatabaseConfirmationPanel.tsx

```diff
@@ -0,0 +1,187 @@
+import { useBoolean } from "@uifabric/react-hooks";
+import { Text, TextField } from "office-ui-fabric-react";
+import React, { FunctionComponent, useState } from "react";
+import LoadingIndicator_3Squares from "../../../images/LoadingIndicator_3Squares.gif";
+import * as Constants from "../../Common/Constants";
+import { deleteDatabase } from "../../Common/dataAccess/deleteDatabase";
+import DeleteFeedback from "../../Common/DeleteFeedback";
+import { getErrorMessage, getErrorStack } from "../../Common/ErrorHandlingUtils";
+import * as ViewModels from "../../Contracts/ViewModels";
```

Comment:
Similar to my comments above, we only use `ViewModels.Collection` and `ViewModels.Database` here. We should only import the ones we need.

Reply:
Updated